src/components/ArchiveSidebar.svelte

```diff
@@ -0,0 +1,85 @@
+<script>
+  import { fly } from "svelte/transition";
+  import {
+    Col,
+    Container,
+    Row,
+    Card,
+    CardHeader,
+    CardBody,
+    Icon,
+  } from "sveltestrap/src";
+  import { stores } from "@sapper/app";
+
+  import ViewCard from "./ViewCard.svelte";
+
+  const { session } = stores();
+
+  export let id;
+  export let is_archive_sidebar_shown;
+  export function closeSidebar() {
+    is_archive_sidebar_shown = false;
+  }
```

Comment:
Remove `export` since functions cannot be exported in Svelte.

Also put a newline between the `export let`s and this function declaration like so:

```js
export let id;
export let is_archive_sidebar_shown;

function closeSidebar() {
  ...
}
```